libs/designer/src/lib/ui/templates/cards/templateCard.tsx

```diff
@@ -81,33 +98,39 @@ export const TemplateCard = ({ templateName }: TemplateCardProps) => {
     root: { display: 'inline-block', maxWidth: 1000 },
   };
 
+  const isMicrosoftAuthored = equals(details?.By, 'Microsoft');
+
   return (
     <DocumentCard className="msla-template-card-wrapper" styles={cardStyles} onClick={onSelectTemplate} aria-label={title}>
-      <div className="msla-template-card-data">
-        <Text size={400} weight="semibold" align="start" className="msla-template-card-title">
-          {title}
-        </Text>
-        <div className="msla-template-card-tags">
-          {['By', 'Type', 'Trigger'].map((key: string) => {
-            if (!details[key]) {
-              return null;
-            }
-            return (
-              <Text key={key} size={300} className="msla-template-card-tag">
-                {key}: {details[key]}
-              </Text>
-            );
-          })}
+      <div className="msla-template-card-authored-wrapper">
+        <div className="msla-template-card-authored">
+          {isMicrosoftAuthored ? <Image src={MicrosoftIcon} alt="Microsoft" width={16} /> : <PeopleCommunity16Regular />}
```

Comment:
Can you use ariaLabel instead of alt? It will take care of accessibility..  we are using fluent components so we don't have to worry about a11y